arrow-core-test/build.gradle

```
@@ -7,11 +7,11 @@ plugins {
 apply from: "$SUB_PROJECT"
 
 dependencies {
-    compile project(":arrow-core")
-    compile project(":arrow-core-data")
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
-    compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
-    compile "io.kotlintest:kotlintest-runner-junit5:$KOTLIN_TEST_VERSION", excludeArrow
-    testRuntime "org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION"
+    api project(":arrow-core")
+    api project(":arrow-core-data")
```

I don't think we need to expose those really

```suggestion
    implementation project(":arrow-core")
    implementation project(":arrow-core-data")
```